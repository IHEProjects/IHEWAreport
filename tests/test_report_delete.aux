\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\abx@aux@refcontext{ynt/global//global/global}
\@writefile{toc}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lot}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{List of Figures}{i}{section*.2}\protected@file@percent }
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{List of Tables}{ii}{section*.3}\protected@file@percent }
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}\protected@file@percent }
\newlabel{sec:Introduction}{{1}{1}{Introduction}{section.1}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Mindanao River Basin\relax }}{1}{figure.caption.4}\protected@file@percent }
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{figure:fig1}{{1}{1}{Mindanao River Basin\relax }{figure.caption.4}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Observation data}{1}{subsection.1.1}\protected@file@percent }
\newlabel{subsec:Observationdata}{{1.1}{1}{Observation data}{subsection.1.1}{}}
\gdef \LT@i {\LT@entry 
    {4}{59.084pt}\LT@entry 
    {4}{96.13907pt}\LT@entry 
    {4}{62.0pt}\LT@entry 
    {1}{75.55495pt}\LT@entry 
    {1}{64.4166pt}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces Discharge at river mouth of Mindanao River\relax }}{2}{figure.caption.5}\protected@file@percent }
\newlabel{figure:fig2}{{2}{2}{Discharge at river mouth of Mindanao River\relax }{figure.caption.5}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {2}RS data analysis}{2}{section.2}\protected@file@percent }
\newlabel{sec:RSdataanalysis}{{2}{2}{RS data analysis}{section.2}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Review of RS products}{2}{subsection.2.1}\protected@file@percent }
\newlabel{subsec:ReviewofRSproducts}{{2.1}{2}{Review of RS products}{subsection.2.1}{}}
\@writefile{lot}{\defcounter {refsection}{0}\relax }\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Remote sensing products\relax }}{2}{table.1}\protected@file@percent }
\newlabel{table:tab1}{{1}{2}{Review of RS products}{table.1}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Precipitation products}{3}{subsection.2.2}\protected@file@percent }
\newlabel{subsec:Precipitationproducts}{{2.2}{3}{Precipitation products}{subsection.2.2}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {3}{\ignorespaces Precipitation products for Mindanao River Basin\relax }}{3}{figure.caption.6}\protected@file@percent }
\newlabel{figure:fig3}{{3}{3}{Precipitation products for Mindanao River Basin\relax }{figure.caption.6}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {4}{\ignorespaces Correlation between precipitation products (unit: mm/month)\relax }}{3}{figure.caption.7}\protected@file@percent }
\newlabel{figure:fig4}{{4}{3}{Correlation between precipitation products (unit: mm/month)\relax }{figure.caption.7}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {5}{\ignorespaces Monthly mean precipitation for Mindanao River Basin\relax }}{4}{figure.caption.8}\protected@file@percent }
\newlabel{figure:fig5}{{5}{4}{Monthly mean precipitation for Mindanao River Basin\relax }{figure.caption.8}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {6}{\ignorespaces Annual precipitation for Mindanao River Basin\relax }}{4}{figure.caption.9}\protected@file@percent }
\newlabel{figure:fig6}{{6}{4}{Annual precipitation for Mindanao River Basin\relax }{figure.caption.9}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {2.3}Evapotranspiration products}{4}{subsection.2.3}\protected@file@percent }
\newlabel{subsec:Evapotranspirationproducts}{{2.3}{4}{Evapotranspiration products}{subsection.2.3}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {7}{\ignorespaces Annual evapotranspiration for Mindanao River Basin\relax }}{5}{figure.caption.10}\protected@file@percent }
\newlabel{figure:fig7}{{7}{5}{Annual evapotranspiration for Mindanao River Basin\relax }{figure.caption.10}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {8}{\ignorespaces Correlation between actual evapotranspiration products (unit: mm/month)\relax }}{5}{figure.caption.11}\protected@file@percent }
\newlabel{figure:fig8}{{8}{5}{Correlation between actual evapotranspiration products (unit: mm/month)\relax }{figure.caption.11}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {9}{\ignorespaces Monthly mean actual evapotranspiration for Mindanao River Basin\relax }}{6}{figure.caption.12}\protected@file@percent }
\newlabel{figure:fig9}{{9}{6}{Monthly mean actual evapotranspiration for Mindanao River Basin\relax }{figure.caption.12}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {10}{\ignorespaces Annual actual evapotranspiration for Mindanao River Basin\relax }}{6}{figure.caption.13}\protected@file@percent }
\newlabel{figure:fig10}{{10}{6}{Annual actual evapotranspiration for Mindanao River Basin\relax }{figure.caption.13}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {2.4}Grace solutions (change in storage)}{6}{subsection.2.4}\protected@file@percent }
\newlabel{subsec:Gracesolutions(changeinstorage)}{{2.4}{6}{Grace solutions (change in storage)}{subsection.2.4}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {11}{\ignorespaces GRACE products for Mindanao River Basin\relax }}{7}{figure.caption.14}\protected@file@percent }
\newlabel{figure:fig11}{{11}{7}{GRACE products for Mindanao River Basin\relax }{figure.caption.14}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {12}{\ignorespaces Correlation between GRACE products (unit: mm/month)\relax }}{7}{figure.caption.15}\protected@file@percent }
\newlabel{figure:fig12}{{12}{7}{Correlation between GRACE products (unit: mm/month)\relax }{figure.caption.15}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {13}{\ignorespaces Monthly mean storage change for Mindanao River Basin\relax }}{8}{figure.caption.16}\protected@file@percent }
\newlabel{figure:fig13}{{13}{8}{Monthly mean storage change for Mindanao River Basin\relax }{figure.caption.16}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {14}{\ignorespaces Annual storage change for Mindanao River Basin\relax }}{8}{figure.caption.17}\protected@file@percent }
\newlabel{figure:fig14}{{14}{8}{Annual storage change for Mindanao River Basin\relax }{figure.caption.17}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {2.5}Runoff comparison}{8}{subsection.2.5}\protected@file@percent }
\newlabel{subsec:Runoffcomparison}{{2.5}{8}{Runoff comparison}{subsection.2.5}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {15}{\ignorespaces Performance of different combinations of the remote sensing products to calculate the runoff generated\relax }}{9}{figure.caption.18}\protected@file@percent }
\newlabel{figure:fig15}{{15}{9}{Performance of different combinations of the remote sensing products to calculate the runoff generated\relax }{figure.caption.18}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {subsection}{\numberline {2.6}Water Balance error}{10}{subsection.2.6}\protected@file@percent }
\newlabel{subsec:WaterBalanceerror}{{2.6}{10}{Water Balance error}{subsection.2.6}{}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {16}{\ignorespaces Water Balance error\relax }}{11}{figure.caption.19}\protected@file@percent }
\newlabel{figure:fig16}{{16}{11}{Water Balance error\relax }{figure.caption.19}{}}
\gdef \LT@ii {\LT@entry 
    {1}{32.02785pt}\LT@entry 
    {4}{37.00002pt}\LT@entry 
    {4}{37.00002pt}\LT@entry 
    {4}{30.33334pt}\LT@entry 
    {1}{74.4856pt}\LT@entry 
    {4}{37.00002pt}\LT@entry 
    {4}{35.33334pt}\LT@entry 
    {1}{36.58276pt}}
\@writefile{lof}{\defcounter {refsection}{0}\relax }\@writefile{lof}{\contentsline {figure}{\numberline {17}{\ignorespaces Yearly mean runoff generation map of the best combination\relax }}{12}{figure.caption.20}\protected@file@percent }
\newlabel{figure:fig17}{{17}{12}{Yearly mean runoff generation map of the best combination\relax }{figure.caption.20}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{\numberline {3}Selection of RS products for WA}{12}{section.3}\protected@file@percent }
\newlabel{sec:SelectionofRSproductsforWA}{{3}{12}{Selection of RS products for WA}{section.3}{}}
\@writefile{lot}{\defcounter {refsection}{0}\relax }\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces The annual $P-ET-\Delta S$ and $\Delta S$, (unit: $Mm^3/year$)\relax }}{12}{table.2}\protected@file@percent }
\newlabel{table:tab2}{{2}{12}{Selection of RS products for WA}{table.2}{}}
\newlabel{sec:References}{{3}{14}{References}{section*.21}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{References}{14}{section*.21}\protected@file@percent }
\newlabel{sec:Annexes}{{3}{15}{Annexes}{section*.22}{}}
\@writefile{toc}{\defcounter {refsection}{0}\relax }\@writefile{toc}{\contentsline {section}{Annexes}{15}{section*.22}\protected@file@percent }
\newlabel{LastPage}{{}{15}{}{page.15}{}}
\xdef\lastpage@lastpage{15}
\xdef\lastpage@lastpageHy{15}
