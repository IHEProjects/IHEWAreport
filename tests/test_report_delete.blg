[0] Config.pm:304> INFO - This is Biber 2.14
[0] Config.pm:307> INFO - Logfile is 'test_report_delete.blg'
[68] biber-MSWIN64:322> INFO - === 
[87] Biber.pm:415> INFO - Reading 'test_report_delete.bcf'
[181] Biber.pm:943> INFO - Found 0 citekeys in bib section 0
[187] Utils.pm:304> WARN - The file 'test_report_delete.bcf' does not contain any citations!
[197] bbl.pm:652> INFO - Writing 'test_report_delete.bbl' with encoding 'UTF-8'
[197] bbl.pm:755> INFO - Output to test_report_delete.bbl
[197] Biber.pm:128> INFO - WARNINGS: 1
