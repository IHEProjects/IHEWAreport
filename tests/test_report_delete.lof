\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax 
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {1}{\ignorespaces Mindanao River Basin\relax }}{1}{figure.caption.4}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {2}{\ignorespaces Discharge at river mouth of Mindanao River\relax }}{2}{figure.caption.5}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {3}{\ignorespaces Precipitation products for Mindanao River Basin\relax }}{3}{figure.caption.6}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {4}{\ignorespaces Correlation between precipitation products (unit: mm/month)\relax }}{3}{figure.caption.7}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {5}{\ignorespaces Monthly mean precipitation for Mindanao River Basin\relax }}{4}{figure.caption.8}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {6}{\ignorespaces Annual precipitation for Mindanao River Basin\relax }}{4}{figure.caption.9}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {7}{\ignorespaces Annual evapotranspiration for Mindanao River Basin\relax }}{5}{figure.caption.10}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {8}{\ignorespaces Correlation between actual evapotranspiration products (unit: mm/month)\relax }}{5}{figure.caption.11}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {9}{\ignorespaces Monthly mean actual evapotranspiration for Mindanao River Basin\relax }}{6}{figure.caption.12}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {10}{\ignorespaces Annual actual evapotranspiration for Mindanao River Basin\relax }}{6}{figure.caption.13}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {11}{\ignorespaces GRACE products for Mindanao River Basin\relax }}{7}{figure.caption.14}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {12}{\ignorespaces Correlation between GRACE products (unit: mm/month)\relax }}{7}{figure.caption.15}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {13}{\ignorespaces Monthly mean storage change for Mindanao River Basin\relax }}{8}{figure.caption.16}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {14}{\ignorespaces Annual storage change for Mindanao River Basin\relax }}{8}{figure.caption.17}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {15}{\ignorespaces Performance of different combinations of the remote sensing products to calculate the runoff generated\relax }}{9}{figure.caption.18}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {16}{\ignorespaces Water Balance error\relax }}{11}{figure.caption.19}%
\defcounter {refsection}{0}\relax 
\contentsline {figure}{\numberline {17}{\ignorespaces Yearly mean runoff generation map of the best combination\relax }}{12}{figure.caption.20}%
