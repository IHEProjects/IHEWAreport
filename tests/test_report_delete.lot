\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax 
\defcounter {refsection}{0}\relax 
\contentsline {table}{\numberline {1}{\ignorespaces Remote sensing products\relax }}{2}{table.1}%
\defcounter {refsection}{0}\relax 
\contentsline {table}{\numberline {2}{\ignorespaces The annual $P-ET-\Delta S$ and $\Delta S$, (unit: $Mm^3/year$)\relax }}{12}{table.2}%
