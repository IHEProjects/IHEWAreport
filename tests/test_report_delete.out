\BOOKMARK [1][]{section*.1}{Contents}{}% 1
\BOOKMARK [1][]{section*.2}{List of Figures}{}% 2
\BOOKMARK [1][]{section*.3}{List of Tables}{}% 3
\BOOKMARK [1][]{section.1}{Introduction}{}% 4
\BOOKMARK [2][]{subsection.1.1}{Observation data}{section.1}% 5
\BOOKMARK [1][]{section.2}{RS data analysis}{}% 6
\BOOKMARK [2][]{subsection.2.1}{Review of RS products}{section.2}% 7
\BOOKMARK [3][]{subsubsection.2.1.1}{Precipitation products}{subsection.2.1}% 8
\BOOKMARK [3][]{subsubsection.2.1.2}{Evapotranspiration products}{subsection.2.1}% 9
\BOOKMARK [3][]{subsubsection.2.1.3}{Grace solutions \(change in storage\)}{subsection.2.1}% 10
\BOOKMARK [2][]{subsection.2.2}{Runoff comparison}{section.2}% 11
\BOOKMARK [2][]{subsection.2.3}{Water Balance error}{section.2}% 12
\BOOKMARK [1][]{section.3}{Selection of RS products for WA}{}% 13
\BOOKMARK [1][]{section*.21}{References}{}% 14
\BOOKMARK [1][]{section*.22}{Annexes}{}% 15
\BOOKMARK [2][]{subsection.3.1}{Remote sensing products}{section*.22}% 16
\BOOKMARK [2][]{subsection.3.2}{The yearly mean P-ET-S and Q, \(unit: Mm3/year\)}{section*.22}% 17
\BOOKMARK [2][]{subsection.3.3}{The water balance P-ET-S and discharge Q, \(unit: mm/month\)}{section*.22}% 18
